packages/main/src/Dialog.ts

```diff
@@ -608,8 +608,14 @@ class Dialog extends Popup {
 				this._initialLeft! + this._initialWidth!,
 			);
 
+			Object.assign(this.style, {
```

Comment:
add comment like

`// check if width is changed to avoid "left" jumping when max width is reached`